database/sqlite/sqlite_aclk_node.c

```diff
@@ -39,7 +39,7 @@ void sql_build_node_collectors(struct aclk_database_worker_config *wc)
     DICTIONARY *dict = dictionary_create(DICTIONARY_FLAG_SINGLE_THREADED);
 
     upd_node_collectors.node_id = wc->node_id;
-    upd_node_collectors.claim_id = is_agent_claimed();
+    upd_node_collectors.claim_id = get_agent_claimid();
 
     upd_node_collectors.node_collectors = collectors_from_charts(wc->host, dict);
```

Comment:
This last does not have correlation with this PR, but I also received it:

```sh
*** CID 355299:  Null pointer dereferences  (NULL_RETURNS)
/database/sqlite/sqlite_aclk_node.c: 45 in sql_build_node_collectors()
39         DICTIONARY *dict = dictionary_create(DICTIONARY_FLAG_SINGLE_THREADED);
40     
41         upd_node_collectors.node_id = wc->node_id;
42         upd_node_collectors.claim_id = get_agent_claimid();
43     
44         upd_node_collectors.node_collectors = collectors_from_charts(wc->host, dict);
>>>     CID 355299:  Null pointer dereferences  (NULL_RETURNS)
>>>     Dereferencing a pointer that might be "NULL" "upd_node_collectors.claim_id" when calling "aclk_update_node_collectors".
45         aclk_update_node_collectors(&upd_node_collectors);
46     
47         dictionary_destroy(dict);
48         freez(upd_node_collectors.claim_id);
49     
50         log_access("ACLK RES [%s (%s)]: NODE COLLECTORS SENT", wc->node_id, wc->host->hostname);
```

If this last was already addressed with other PR, probably we need to rebase this one.